Cargo.toml

```diff
@@ -36,11 +37,16 @@ tempfile = "3.8"
 arrow = { version = "50.0", default-features = false, features = ["ffi"] }
 
 [workspace]
-members = ["gdal-sys"]
+members = ["gdal-src", "gdal-sys"]
 
 # docs.rs-specific configuration
 [package.metadata.docs.rs]
 # include `array` feature in documentation
 features = ["array"]
 # define attribute `docsrs` for feature badges
 rustdoc-args = ["--cfg", "docsrs"]
+
+
+[patch.crates-io]
+proj-sys = { git = "https://github.com/GiGainfosystems/proj", rev = "44a503869709ea793a5f8452bfb9e2a99b7160fc" }
+pq-src = { git = "https://github.com/sgrif/pq-sys", rev = "7300e96574cc0eeb95985881fbbc2cf95886cca2" }
```

Comment:
I found another bug in the pg-src crate while setting up the CI here. I will issue an release in the next few days and update this PR afterwards to depend on the fixed version instead

Reply:
Switched to a release version in https://github.com/georust/gdal/pull/517/commits/cf39e1ff1e9fce2a59df334b9e4344e0231ee9f7